07-reputation/04-globaltrust.md

```diff
@@ -0,0 +1,39 @@
+## Global Trust
+
+*Global trust*
+: the result of the [EigenTrust](http://ilpubs.stanford.edu:8090/562/1/2002-56.pdf) algorithm -- the trust in the network participant, which was obtained taking into account *all* local trusts of *all* nodes.
```

Comment:
... algorithm is the trust in the network participant, which has been obtained regarding *all* local trusts of *all* nodes.

Reply:
fixed